chainerrl/agents/double_iqn.py

```diff
@@ -0,0 +1,65 @@
+from __future__ import unicode_literals
+from __future__ import print_function
+from __future__ import division
+from __future__ import absolute_import
+from future import standard_library
+standard_library.install_aliases()  # NOQA
+
+import chainer
+import chainer.functions as F
+
+from chainerrl.agents import iqn
+
+
+class DoubleIQN(iqn.IQN):
+    """Double IQN - Use primary network for target computation."""
```

Comment:
I don't think `Use primary network for target computation` is a good description of the algorithm. Can you just say something like `IQN with DoubleDQN-like target computation` and add a detailed explanation after a blank line (as suggested by the google style https://github.com/google/styleguide/blob/gh-pages/pyguide.md#38-comments-and-docstrings)?